README.md

```diff
@@ -107,6 +106,23 @@ the login script to refresh your session to be able to see the updates.
 
 ## Playbooks
 
+### Portal Ping
+
+#### Playbook Actions
+This playbook pings a portal to check if it is accessible. 
+
+**NOTE** this playbook is a wrapper for the `ping` command line tool and uses
```

Comment:
It's a wrapper for Ansible `ping` module, not `ping` command line tool.

`ping` CLI tool might be ok (IP of the server is accessible), but the portal might not be pingable from Ansible.